src/components/inspectors/WeekdaySelect.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div v-if="cycleManager.isWeeklyPeriodSelected()" class="mb-2">
+    <label class="">{{ $t(weekLabel) }}</label>
+    <div>
+      <span
+        v-for="day in weekdays"
+        :key="day.day"
+        class="badge badge-pill weekday mb-1"
+        :class="cycleManager.weekdayStyle(day)"
+        :data-test="`day-${ day.day }`"
+        @click="clickWeekDay(day)"
+      >
+        {{ $t(day.initial) }}
+      </span>
+    </div>
+    <small v-if="repeatOnValidationError" class="text-danger">{{ repeatOnValidationError }}</small>
+    <small class="form-text text-muted">{{ $t(periodicityHelper) }}</small>
+  </div>
+</template>
+
+<script>
+import { DateTime } from 'luxon';
+import CycleManager from '@/components/inspectors/CycleManager';
+
+export default {
+  props: {
+    selectWeekdays: {
+      type: Array,
+      default: () => [],
+    },
+    startDate: {
+      type: String,
+      default: DateTime.local().toUTC().toISO().toString(),
```

Comment:
[`toISO`](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-toISO) already returns a string so I think calling `toString` on it is redundant.
```suggestion
      default: DateTime.local().toUTC().toISO(),
```